### PR TITLE
Set clientSideOnly = true

### LINKED ISSUE #16: Clientside code on the server - java.lang.NoClassDefFoundError: santa/ping/PingSound
https://paste.fedoraproject.org/paste/2wiZh75PZOrPsk1KZTEKmQ

If it isn't needed by the server, set it as a clientside mod so Forge doesn't load it on a dedicated server?

* [ ] 1.7
* [ ] 1.8
* [x] 1.9

### PR DESCRIPTION
**Pull request by mezz**
Fixes #16
This flag has been available since at least 1.8 so it should be safe to set for all the versions this mod supports for 1.9+.
It will make the mod do nothing on the server, FML will just skip loading it.

### CODE REVIEW

File: src/main/scala/santa/ping/Ping.scala [hunk 1]
```diff
@@ -19,4 +19,4 @@ object Ping {
   def init(event: FMLInitializationEvent): Unit = {
     MinecraftForge.EVENT_BUS.register(new PingEventHandler())
   }
-}
```
Comment (elifoster) [inline src/main/scala/santa/ping/Ping.scala]: Can you undo this change? I'd rather keep this PR only changing the Mod annotation

**Review by elifoster — CHANGES_REQUESTED** [review]

File: src/main/scala/santa/ping/Ping.scala (hunk 1 above)
Comment (mezz) [reply src/main/scala/santa/ping/Ping.scala]: Sure

Comment (mezz) [reply src/main/scala/santa/ping/Ping.scala]: Seems github editor does it automatically and wont let me put it back.

Comment (mezz) [reply src/main/scala/santa/ping/Ping.scala]: Done

**Review by elifoster — APPROVED** [review]
